SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -2204,6 +2204,7 @@
                 "gtk-update-icon-cache",
                 "helm",
                 "hvloader",
+                "ig",
```

Comment:
Similar question as for #8177 against `main`: why do we want to add this package?

Reply:
We have a workitem for it: [link](We do have a workitem for it: [link](https://dev.azure.com/microsoft/OS/_workitems/edit/49010795))

Reply:
We do have a workitem for it: [link](https://dev.azure.com/microsoft/OS/_workitems/edit/49010795)